core/controllers/skill_mastery.py

```diff
@@ -173,9 +172,8 @@ def get(self):
                 if skill_mastery_dict:
                     # Subtopic mastery is average of skill masteries.
                     subtopic_mastery_dict[topic.id][subtopic.id] = (
-                        python_utils.divide(
-                            sum(skill_mastery_dict.values()),
-                            len(skill_mastery_dict)))
+                        sum(skill_mastery_dict.values()) // len(
+                            skill_mastery_dict))
```

Comment:
`skill_mastery_dict.values()` is float, shoudl we use `/`?

Reply:
> `skill_mastery_dict.values()` is float, shoudl we use `/`?

How should I know about this function is float or not?? I searched it in the codebase but not able to find it.